frontend/app/routes/public/renew/$id/ita/review-information.tsx

```diff
@@ -403,11 +403,11 @@ export default function RenewItaReviewInformation() {
             variant="green"
             disabled={isSubmitting}
             loading={isSubmitting && submitAction === FormAction.Submit}
-            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult:Submit - Review your information click"
+            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-ITA:Submit - Review information click"
```

Comment:
(Same comment for the rest of the attribute values in this route)

```suggestion
            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-ITA:Submit - Review your information click"
```